src/main/webapp/data/userMapData.json

```diff
@@ -1643,7 +1677,9 @@
       "Northwood University",
       "Norwich Technical High School",
       "Nyack High School",
+      "Oakland University",
       "Oakton Community College",
+      "Oak Creek High School",
```

Comment:
Should we put this line before `Oakland University`?

Reply:
Ok I think for now it is enough for us to put this entry in front. On a side note, I'm not sure about how you actually compare the two entries, but in my view, even comparing `Oakton...` and `OakCreek...` should put `OakCreek...` in front.